otj-metrics-actuate/src/main/java/com/opentable/metrics/actuate/micrometer/OtMicrometerToDropWizardExportConfiguration.java

```diff
@@ -42,29 +49,38 @@ public String prefix() {
 
             @Override
             @Nullable
+            // Why is this returning null?
             public String get(String s) {
                 return null;
             }
         };
     }
 
-
+    /**
+     * Converts from dimensional tags (eg prometheus, micrometer) to graphite (flat)
+     * DMITRY: Please comment and explain. Also what's wrong with HierarchicalNameMapper.default
+     */
```

Comment:
1) I wanted to add V2 prefix
2) HierarchicalNameMapper.default adding tag names, which makes metrics less readabale
3) It was generating some incompatible names (as I remember)